governance.md

```diff
@@ -1,98 +1,137 @@
-The committee may choose to make this a permanent role at their discretion.
-Deliverables may include creation of mailing lists, project tracking boards, governance draft documents, and so forth.
+* The [Code of Conduct](code-of-conduct.md) and handling violations
+* The [Project Communincation Channels](contact.md)
+* The [Contribution Process](process.md) and [Development Standards](standards.md)
+* The [Tekton Mission and Vision](roadmap.md)
+* Select [election officers](#election-officers) to run elections
 
-## Committee Deliverables
+More?
 
-The committee will be responsible for a series of specific artifacts and activities as outlined below.
+## Governance Meetings and Decision-Making Process
 
-### Initial Charter
+Governance decisions, votes and questions should take place on the
+tekton-governance@googlegroups.com mailing list.
 
-This document will define how the committee is to manage the project until it has transitioned to an elected steering body, as well as what governance must be in place.
-The Kubernetes Steering Committee Charter Draft serves as a good example.
+TBD
 
-A charter should cover all of the following topics:
+## Elections
 
-* Scope of rights and responsibilities explicitly held by the committee
-* Committee structure that meets the requirements above
-* Election process, including:
-  * special elections in the case someone resigns or is impeached
-  * who is eligible to nominate candidates and how
-  * who is eligible to run as a candidate
-  * Voter registration and requirements
-  * election mechanics such as
-    * committee company representation quotas
-    * Limits on electioneering
-    * Responses to election fraud
-  * How are changes to the charter enacted, and by what process
-  * How are meetings conducted
-    * Recorded or not, and if not, how is the information shared
-    * How is work tracked? Example steering project board
-    * Is there a member note taker, or is there a neutral facilitator role that exists outside of the committee?
-    * Frequency, duration, and required consistency
-  * Committee decision-making process, and specifically those areas of action that require more/less consensus, e.g. modifications the charter
-  * Sub-Steering Committee governance structure (see this example)
+### Voter Eligibility
 
-## Transition Process
+Anyone who has at least 15 contributions in the last 12 months. Contributions
+include opening PRs, reviewing and commenting on PRs, opening and commenting on
+issues, writing design docs, commenting on design docs, helping people on slack,
+participating in working groups. The
+[dashboard on tekton.devstats.cd.foundation](https://tekton.devstats.cd.foundation/d/9/developer-activity-counts-by-repository-group-table?orgId=1&var-period_name=Last%20year&var-metric=contributions&var-repogroup_name=All&var-country_name=All)
+will show GitHub based contributions that are > 20 total.
```

Comment:
I think thanks to https://github.com/cncf/devstats/issues/237 we can now see all contributors in this view (or at least it isnt limited to 20 now)

Reply:
indeed